src/native/api.c

```diff
@@ -260,7 +324,7 @@ int llhttp__on_chunk_complete(llhttp_t* s, const char* p, const char* endp) {
 }
 
 
-/* Private */
+/* Private https://github.com/dnlup/llhttp/blob/undici_wasm/wasm.js*/
```

Comment:
There are no changes below. Is this comment necessary?

Reply:
Nope, sorry.